Client.UnitTests/TestDataProvider.cs

```diff
@@ -143,6 +143,15 @@ public static IEnumerable<TestCaseData> Provider()
                 new EvaluateDecisionResponse(),
                 (RequestCreator<IEvaluateDecisionResponse>)
                 (zeebeClient => zeebeClient.NewEvaluateDecisionCommand().DecisionId("decision")));
+            yield return new TestCaseData(
+                new UpdateJobTimeoutRequest()
```

Comment:
👍🏼 Great that you added here an test case as well.

As this is only for the SendWithRetry test right now we also need some more tests for the happy path to check the expected requests, and responses. Like we do with the other commands. Take as example this https://github.com/camunda-community-hub/zeebe-client-csharp/blob/main/Client.UnitTests/UpdateRetriesTest.cs
What we also need is to add

Reply:
I added tests to check expected requests